ui/app/src/state/reducers/preview.ts

```diff
@@ -32,9 +32,9 @@ import {
   SELECT_FOR_EDIT,
   SELECT_PREVIOUS_TOOL,
   SELECT_TOOL,
-  SET_AUDIENCES_PANEL_MODEL,
-  SET_AUDIENCES_PANEL_MODEL_COMPLETE,
-  SET_AUDIENCES_PANEL_MODEL_FAILED,
+  SET_ACTIVE_PROFILE,
+  SET_ACTIVE_PROFILE_COMPLETE,
+  SET_ACTIVE_PROFILE_FAILED,
```

Comment:
I think you've updated all over to be model instead of profile; let's update these too so naming is consistent *eeeeverywhere*